### PR TITLE
Implement logging for PromethION script

### PR DESCRIPTION
**Pull request by kedhammar**
Tested on prom.

### CODE REVIEW

**Comment by kedhammar** [comment]
TODO add `--log_path "/data/promethion_transfer_log.txt"` to crontab cmd and AM doc

File: taca/nanopore/promethion_transfer.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 """ Transfers new PromethION runs to ngi-nas using rsync.
 """
-__version__ = "1.0.8"
+__version__ = "1.0.10"
```
Comment (ssjunnebo) [inline taca/nanopore/promethion_transfer.py:3]: Why are we skipping 9?

**Review by ssjunnebo — APPROVED** [review]
Very nice! 👏

File: taca/nanopore/promethion_transfer.py (hunk 1 above)
Comment (kedhammar) [reply taca/nanopore/promethion_transfer.py:3]: Because seven ate nine

**Pull request merged**